### PR TITLE
doc: release notes: RISC-V release notes for 2.0

### PR DESCRIPTION
**Pull request by nategraff-sifive**
Adds 2.0 release notes for the RISC-V architecture.

### CODE REVIEW

File: doc/releases/release-notes-2.0.rst [hunk 1]
```diff
@@ -86,6 +86,16 @@ Architectures
     MSI and other features required for PCIe devices. The previous PCI
     implementation has been deprecated and will be removed in 2.1.
 
+* RISC-V:
+
+  * Added support for the SiFive HiFive1 Rev B development board.
+  * Added support for LiteX VexRISCv soft core.
+  * Added support for 64-bit RISC-V, renaming the architecture from "riscv32"
+    to "riscv".
+  * Added 64-bit Qemu support.
```
Comment (dbkinder) [inline doc/releases/release-notes-2.0.rst]: ``QEMU``

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-2.0.rst (hunk 1 above)
Comment (nategraff-sifive) [reply doc/releases/release-notes-2.0.rst]: Do you want me to fix the one in ARM's release notes too?

Comment (dbkinder) [reply doc/releases/release-notes-2.0.rst]: Yes, please!

Comment (nategraff-sifive) [reply doc/releases/release-notes-2.0.rst]: Done!

**Review by dbkinder — APPROVED** [review]
Thanks!

**Review by ioannisg — APPROVED** [review]
Thank you

**Review by aescolar — APPROVED** [review]

**Review by kgugala — APPROVED** [review]
LGTM

**Pull request merged**